### PR TITLE
Add Data

### PR DESCRIPTION
**Pull request by Nolle10**

### CODE REVIEW

File: CarboPlanner/src/views/Loading.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default function Loading({route}) {
       let listOfVolumes = new Map<string, number>;
 
         // TODO change to value when API is fixed
```
Comment (frmad) [inline CarboPlanner/src/views/Loading.tsx]:
```suggestion
       
```

File: CarboPlanner/src/views/Loading.tsx [hunk 2]
```diff
@@ -47,19 +47,9 @@ export default function Loading({route}) {
             }
         }
         // TODO change to value when API is fixed
```
Comment (frmad) [inline CarboPlanner/src/views/Loading.tsx]:
```suggestion
       
```

File: CarboPlanner/src/views/Loading.tsx [hunk 3]
```diff
@@ -179,9 +169,9 @@ export default function Loading({route}) {
     //const { base64 } = route.params;
     const { firstImageBase64, allImagesBase64 } = route.params;
 
-    //const api_url = process.env.EXPO_PUBLIC_API_URL;
+    const api_url = process.env.EXPO_PUBLIC_API_URL;
 
-    const api_url : string = 'https://yolov5-flaskapi-5qhj5kt2ta-lz.a.run.app/v1/object-detection';
+    //const api_url : string = 'https://yolov5-flaskapi-5qhj5kt2ta-lz.a.run.app/v1/object-detection';
     //GAMLE - https://yolov5-flaskapi-5qhj5kt2ta-lz.a.run.app/v1/object-detection
```
Comment (frmad) [inline CarboPlanner/src/views/Loading.tsx]:
```suggestion
   
```

File: CarboPlanner/src/views/Loading.tsx [hunk 4]
```diff
@@ -179,9 +169,9 @@ export default function Loading({route}) {
     //const { base64 } = route.params;
     const { firstImageBase64, allImagesBase64 } = route.params;
 
-    //const api_url = process.env.EXPO_PUBLIC_API_URL;
+    const api_url = process.env.EXPO_PUBLIC_API_URL;
 
-    const api_url : string = 'https://yolov5-flaskapi-5qhj5kt2ta-lz.a.run.app/v1/object-detection';
+    //const api_url : string = 'https://yolov5-flaskapi-5qhj5kt2ta-lz.a.run.app/v1/object-detection';
```
Comment (frmad) [inline CarboPlanner/src/views/Loading.tsx:174]:
```suggestion
    
```

**Review by frmad — CHANGES_REQUESTED** [review]

**Review by Sindahlof — APPROVED** [review]
Looks good

**Review by Sindahlof — APPROVED** [review]
looks good

**Review by frmad — APPROVED** [review]
looks fine

**Pull request merged**